CHANGELOG

```diff
@@ -1,3 +1,14 @@
+* Tue Aug 03 2021 Trevor Vaughan <tvaughan@onyxpoint.com> - 4.10.0
+- Fixed
+  - simplib::cron::hour_entry now supports ranges
```

Comment:
Didn't it already support ranges?  Looks what was added was support for comma-separated lists

Reply:
@lnemsick-simp dashes are ranges, commas are individual time entries.

Reply:
@lnemsick-simp Apologies, you're absolutely right, I'll update the comment.